imports/plugins/core/orders/server/no-meteor/queries/refunds.js

```diff
@@ -0,0 +1,50 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import { getPaymentMethodConfigByName } from "/imports/plugins/core/payments/server/no-meteor/registration";
+import { getOrderQuery } from "../util/getOrderQuery";
+
+/**
+ * @name refunds
+ * @method
```

Comment:
How/why are we using `@method` here? I see we're using it all over the place, it's just not clear from the docs what it's supposed to convey or how it renders in the doc site.

Reply:
This is a habit carried over from using `meteor` methods. For certain methods, they wouldn't appear in jsdoc without this. We can probably stop using it, unless anyone has any objections / reasons why we do still use it? @machikoyasuda @mikemurray any ideas?

Reply:
As long as the method's docs appear in http://api.docs.reactioncommerce.com/, we're good 👍 

Back in the old days, some Meteor methods would not appear w/o the `@method`. You can test it out locally by running `npm run docs` and checking the HTML files that get created in your personal `tmp/reaction-docs` folder to see if that method was rendered.